core-bundle/contao/themes/flexible/styles/layout/formbody.pcss

```
@@ -12,6 +12,11 @@
 .tl_submit_container {
     padding: 8px 12px;
     background: var(--panel-bg);
+    border-radius: 0 0 var(--border-radius) var(--border-radius);
+}
+
+.tl_formbody_submit.is-stuck .tl_submit_container {
```

```suggestion
.tl_formbody_submit.is--stuck .tl_submit_container {
```
See above (renaming)